posts/2022/03/17/summarizing-important-cases.mdx

```diff
@@ -0,0 +1,80 @@
+
+Yep.
+
+Altogether, we've gathered over a million summaries like the ones above, and we've associated them with 370,000 opinions. Every day we add more.
+
+
+## How is this possible?
+
+This system works by identifying legal citations followed by court-written summaries in parentheses.
+
+For example, the first summary above is drawn from a 9th Circuit case that has the following footnote:
+
+<CaptionedImage imgProps={{
+  src: "/images/strickland-summary.png",
+  alt: "A screenshot of a summary Strickland, as quoted above."
+}}><em>Strickland</em> is summarized as above in a footnote of <a href="https://www.courtlistener.com/opinion/4474583/joseph-rice-v-w-montgomery/">this 9th Circuit case</a>.</CaptionedImage>
+
+Our process begins by using our [citation parser, eyecite][e], to identify these summaries. Once that's completed, we filter out bad ones, organize good ones into semantically similar groups, and rank the groups from best to worst.
+
+The way we group and rank summaries takes into account a number of factors including the size of the semantic group, the importance of the summarizing opinion, and the text of the summary itself. Between all this, we're able to identify the top summary for each group and the top groups for each opinion.
+
+As you can see above, because these summaries are written by the courts, they tend to be exceptionally accurate. With this feature launched, hundreds of thousands of opinions in CourtListener now have expertly-written summaries.
+
+So that's how we did it, but no project like this happens in a vacuum. We could never have done this without the pioneering example of [Casetext][ct] (where this idea was first put into action), the tireless energy of [Faiz Surani][f] and [Varun Iyer][v] at University of California, Santa Barbara, who did all the heavy lifting in this project, and the [Library Innovation Lab at Harvard][lil], with whom we collaborated on [eyecite][e].
+
+
+## What's next?
+
+This new feature is so powerful this has to be just the beginning. We have two more major components in the works.
+
+First, we plan to export this data into a CSV so that researchers can work with it. That unique new legal resource should be available in the coming days.
+
+Second, we're working on making this data searchable. Instead of searching the full text of opinions, just search their summaries and find the ones that are on point!
+
+If you think you might have ideas for how to use this data or how to make it searchable — or something else we haven't thought of — please get in touch! We're thrilled to take the next step on this journey.
+
+
+[cu]: https://www.courtlistener.com/opinion/1741/citizens-united-v-federal-election-comn/summaries/
+[strickland]: https://www.courtlistener.com/opinion/111170/strickland-v-washington/summaries/
+[e]: /projects/eyecite/
+[f]: https://faizsurani.com/
+[v]: https://linkedin.com/m/varun-iyer
```

Comment:
I got my own linkedin profile wrong :| https://www.linkedin.com/in/varun-m-iyer/